test/unit/db/seeds_test.rb

```diff
@@ -3,6 +3,13 @@
 require 'test_helper'
 
 class SeedsTest < ActiveSupport::TestCase
+  setup do
+    # some annoying test is failing to clear the db in the teardown making this one to randomly fail, so adding this to ensure the test database is "virgin"
+    DatabaseCleaner.clean
```

Comment:
We should remove this now 😄 With the `disable_transactional_fixtures!` added below should be enough

Reply:
Done